tests/bases_fixtures.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[7]:
+
+
+from __future__ import print_function
+import mbuild as mb
+import builtins as __builtin__
+import numpy as np
+from foyer import Forcefield
+# All the old import from compound
+# import collections
+# from collections import OrderedDict, defaultdict
+from copy import deepcopy
+import itertools
+import pytest
+import os
+import sys
+import tempfile
+from warnings import warn
+##
+from mbuild.utils.io import run_from_ipython, import_
+import pytest
+
+
+def ona_box(sequence, dim):
+    class ONA_box(mb.Compound):
```

Comment:
Is this the same `ONA_box` class in `mbuild_ona.py`? If so you can just import it at the top like `from mbuild_ona import ONA_box`.

Reply:
What do you think is a better approach here - remove `ONA_box` from `mbuild_ona.py` and have it focus on the single-chain `DNA`, and let the user create the multi-chain box themselves if they need to, or keep both classes in the plugin, and have entry points for both?

Reply:
So other people in Jayaraman group (who actually have used this model) advised me that in general when they start a simulation with this model, they in general start with double strands of these DNA chains, since it will be very hard to go the other way - have single-strand chains and let them fully and correctly hybridize in the simulation. From this perspective, on top of single strand DNA, maybe we should actually offer the feature of creating a box with properly hybridized strands in the plugin?

Reply:
For the first question, I think it's still worthwhile to keep `ONA_box` and have entry points for both.

Reply:
For your second question, this is good information to know.  I think you can then make `ONA_box` more flexible by allowing the user to specify if they want to initialize with single or double strands of DNA.  Since most of the users in your group are starting with double strands, it makes sense to make this the default argument.  Something like:

```
def __init__(self,sequences = [],dim = [0,0,0], dna_type='double'):
```